src/bp/ui-studio/src/web/views/FlowBuilder/sidePanelFlows/FlowNameModal.tsx

```diff
@@ -94,7 +102,7 @@ const FlowNameModal: FC<Props> = props => {
               type="submit"
               text={submitText}
               onClick={submit}
-              disabled={!name || isIdentical || alreadyExists}
+              disabled={!name || isIdentical || alreadyExists || hasIllegalWord}
```

Comment:
This will never be used, there's a limited character set available for flow name

Reply:
just saw the sanitize name function, gonna remove the check

Reply:
Check and error message translations removed